CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the z/OS FTP Plug-in for Zowe CLI will be documented in this file.
 
+## Recent Changes
+
+- Refine messages and change `bright` command to `zowe`.
```

Comment:
Refine error message of uploading partition dataset member
Refine description of parameter `dcb` of uploading sequential dataset

Reply:
Hey @tiantn,
Should we also add the keyword `BugFix:` in front of @std4lqi's suggestions ?
I'm fine with the PR as it is though 😋

Reply:
I updated the CHANGELOG.md to add Qi's suggestions. Thank you @std4lqi and @zFernand0 😊